src/compositemark/boxplot.ts

```diff
@@ -171,6 +173,30 @@ export function normalizeBoxPlot(spec: GenericUnitSpec<Encoding<string>, BoxPlot
       }
     ]
   };
+
+  if (!isMinMax) {
+    // add outliers
+    const upperBoxField: string = 'upper_box_' + continuousAxisChannelDef.field;
+    const lowerBoxField: string = 'lower_box_' + continuousAxisChannelDef.field;
+    result.layer.push({
```

Comment:
if you just push a layer, this will be incorrect as it takes the same aggregated data source. 

```
data: original data source
layer:
  layer 
     ...all other layers
  new unit spec for outlier
```

Reply:
Do I need to have the same transform in every non-outlier mark unit spec to calculate (q1, q3, median, iqr, etc)? I also then would need to have a transform with window and filter to calculate q1 and q3 and filter out values for outliers point mark spec?